internal/cluster/validator.go

```diff
@@ -281,6 +287,11 @@ func doesClusterNetworksMatchNetworkType(cluster *common.Cluster) bool {
 	})) && cluster.NetworkType != nil && swag.StringValue(cluster.NetworkType) != models.ClusterNetworkTypeOVNKubernetes
 }
 
+func doesAvailabilityModeMatchNetworkType(cluster *common.Cluster) bool {
+	return swag.StringValue(cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone &&
```

Comment:
@empovit the user will still should be able to specify their own network type. We should not allow only SDN

Reply:
@romfreiman Interesting. Can I have more background on the issue? Where does the requirement come from? The ticket says "_Allow OVN only as NetworkType_". Am I misinterpreting it?

Reply:
yea, sorry for the confusion. Look at the epic